mmdet/models/task_modules/prior_generators/anchor_generator.py

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import warnings
+from typing import List, Optional, Tuple, Union
 
 import numpy as np
 import torch
```

Comment:
`from torch import Tensor` 

Then the type-hint can simple use `Tensor` instead of `torch.Tensor`